test/http-fetcher-test.js

```diff
@@ -44,17 +39,12 @@ suite('http-fetcher-test', () => {
     });
   });
 
-  test('it should handle non-json repsonses', () => {
+  test('it should handle non-json responses', () => {
     fetchMock.restore();
```

Comment:
This restore call can go away now.

Reply:
Or maybe it can’t. It maybe can? Worth a try.

Reply:
it can't because it needs to un-mock the url from the setup.